### PR TITLE
(feat) Langfuse - Add location logging, and add cache_hit to metadata.

### LINKED ISSUE #2937: [Feature]: Log region to Langfuse
### The Feature

For things like Vertex AI, it'd be useful if we logged the region that was used.

### Motivation, pitch

For troubleshooting/checking perf of regions.

### Twitter / LinkedIn details

https://www.linkedin.com/in/davidmanouchehri/

### PR DESCRIPTION
**Pull request by Manouchehri**
Resolves #2937.

### CODE REVIEW

File: litellm/integrations/langfuse.py [hunk 1]
```diff
@@ -299,11 +301,24 @@ def _log_langfuse_v2(
                     else:
                         clean_metadata[key] = value
 
+            api_base = litellm_params.get("api_base", None)
```
Comment (ghost) [inline litellm/integrations/langfuse.py:303]: @Manouchehri is this trying to do something similar to this function - https://github.com/BerriAI/litellm/blob/ef4c85522c001c930f02e2ec2c32dea9a7816b74/litellm/utils.py#L5319

Comment (Manouchehri) [reply litellm/integrations/langfuse.py:303]: Yeah, that might work better. I'm just worried it's going to create a different URL than the one that was actually used if I used `get_api_base` incorrectly. =P

Comment (Manouchehri) [reply litellm/integrations/langfuse.py:303]: Could we merge this as-is for now? It's better than nothing, and I'm having trouble keeping up with merge conflicts. :)

Comment (ishaan-jaff) [reply litellm/integrations/langfuse.py:303]: merging this for now and will refactor to to use the `get_api_base`

**Pull request merged**